testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -0,0 +1,362 @@
+        # ensure that writing of DCD files only occurs with properly
+        # opened files
+        with self.assertRaises(IOError):
+            self.dcdfile_r.write(xyz=np.zeros((3,3)),
+                                 box=np.zeros(6, dtype=np.float64),
+                                 step=0,
+                                 natoms=330,
+                                 charmm=0,
+                                 time_step=22.2,
+                                 ts_between_saves=3,
+                                 remarks='')
+
+    def test_written_dcd_coordinate_data_shape(self):
+        # written coord shape should match for all frames
+        expected = (self.natoms, 3)
+        with DCDFile(self.testfile) as f:
+            if f.n_frames > 1:
+                for frame in f:
+                    xyz = f.read()[0]
+                    assert_equal(xyz.shape, expected)
+            else:
+                xyz = f.read()[0]
+                assert_equal(xyz.shape, expected)
+
+    def test_written_unit_cell(self):
+        # written unit cell dimensions should match for all frames
+        expected = np.array([  0.,   0.,   0.,  90.,  90.,  90.],
+                            dtype=np.float32)
+        with DCDFile(self.testfile) as f:
+            if f.n_frames > 1:
+                for frame in f:
+                    unitcell = f.read()[1]
+                    assert_equal(unitcell, expected)
+            else:
+                unitcell = f.read()[1]
+                assert_equal(unitcell, expected)
+
+    def test_written_num_frames(self):
+        with DCDFile(self.testfile) as f:
+            assert_equal(len(f), self.expected_frames)
+
+    def test_written_seek(self):
+        # ensure that we can seek properly on written DCD file
+        with DCDFile(self.testfile) as f:
+            f.seek(self.seek_frame)
+            assert_equal(f.tell(), self.seek_frame)
+
+    def test_written_zero_based_frames(self):
+        # ensure that the first written DCD frame is 0
+        expected_frame = 0
+        with DCDFile(self.testfile) as f:
+            assert_equal(f.tell(), expected_frame)
+
+    def test_written_remarks(self):
+        # ensure that the REMARKS field *can be* preserved exactly
+        # in the written DCD file
+        with DCDFile(self.testfile) as f:
+            assert_equal(f.remarks, self.expected_remarks)
+
+    def test_written_nsavc(self):
+        # ensure that nsavc, the timesteps between frames written
+        # to file, is preserved in the written DCD file
+        expected = self.dcdfile_r.nsavc
+        actual = DCDFile(self.testfile).nsavc
+        assert_equal(actual, expected)
+
+    def test_written_istart(self):
+        # ensure that istart, the starting timestep, is preserved
+        # in the written DCD file
+        expected = self.dcdfile_r.istart
+        actual = DCDFile(self.testfile).istart
+        assert_equal(actual, expected)
+
+    def test_written_delta(self):
```

Comment:
Good point -- and I'm having trouble with this new test, so will need to investigate.